File: src/main/java/com/atsuishio/superbwarfare/client/model/entity/TurretWreckModel.java
package com.atsuishio.superbwarfare.client.model.entity;

import com.atsuishio.superbwarfare.Mod;
import com.atsuishio.superbwarfare.entity.vehicle.TurretWreckEntity;
import net.minecraft.resources.ResourceLocation;
import software.bernie.geckolib.model.GeoModel;

public class TurretWreckModel extends GeoModel<TurretWreckEntity> {

    @Override
    public ResourceLocation getAnimationResource(TurretWreckEntity entity) {
        return null;
    }

    @Override
    public ResourceLocation getModelResource(TurretWreckEntity entity) {
        return Mod.loc("geo/gun_mu.geo.json");
    }

    @Override
    public ResourceLocation getTextureResource(TurretWreckEntity entity) {
        return Mod.loc("textures/entity/empty.png");
    }
}

File: src/main/kotlin/com/atsuishio/superbwarfare/client/model/entity/TurretWreckModel.kt
package com.atsuishio.superbwarfare.client.model.entity

import com.atsuishio.superbwarfare.Mod.Companion.loc
import com.atsuishio.superbwarfare.entity.vehicle.TurretWreckEntity
import software.bernie.geckolib.model.GeoModel

class TurretWreckModel : GeoModel<TurretWreckEntity>() {
    override fun getAnimationResource(entity: TurretWreckEntity) = null

    override fun getModelResource(entity: TurretWreckEntity) = loc("geo/gun_mu.geo.json")

    override fun getTextureResource(entity: TurretWreckEntity) = loc("textures/entity/empty.png")
}
